### PR TITLE
Typo in Const.java #5323

### LINKED ISSUE #5323: Typo in Const.java
line 42 ADMIN_TIMZE_ZONE_DOUBLE should be ADMIN_TIME_ZONE_DOUBLE

### PR DESCRIPTION
**Pull request by gableh**
Fixes #5323

Typos and references updated:

ADMIN_TIMZE_ZONE_DOUBLE -> ADMIN_TIME_ZONE_DOUBLE
ADMIN_EMAIL_RECEVIER -> ADMIN_EMAIL_RECEIVER
ADMIN_EMAIL_ADDRESS_RECEVIERS -> ADMIN_EMAIL_ADDRESS_RECEIVERS			AUTOMATED_LOG_COMILATION -> AUTOMATED_LOG_COMPILATION

### CODE REVIEW

**Comment by wkurniawan07** [comment]
Thanks for the pull request, @gableh!
However, your PR title and description are not conforming to the required format. Do read up our development process as lined out [here](https://github.com/TEAMMATES/teammates/blob/master/devdocs/process.md#fixing-issues), and rename the PR title and description accordingly.

**Comment by gableh** [comment]
Oh i see! Alright, thanks!

**Comment by wkurniawan07** [comment]
@gableh, remember to update the PR title also. It should exactly be the same as the issue title, along with the issue number as well.

**Comment by chowyb** [comment]
Title looks okay now @gableh @wkurniawan07

**Comment by wkurniawan07** [comment]
@gableh can you sync with `master` branch and resolve the conflicts, so that this can be merged?

File: src/main/java/teammates/common/util/ActivityLogEntry.java [hunk 1]
```diff
@@ -18,9 +18,9 @@
 
 /** A log entry to describe an action carried out by the app */
 public class ActivityLogEntry {
-    
+
```
Comment (wkurniawan07) [inline src/main/java/teammates/common/util/ActivityLogEntry.java]: @gableh can you revert changes to the whitespace like this?

File: src/main/java/teammates/logic/automated/AdminEmailWorkerServlet.java [hunk 2]
```diff
@@ -25,19 +25,19 @@
...
     public void doGet(HttpServletRequest req, HttpServletResponse resp) {
-        
-        
+
+
         String emailId = HttpRequestHelper.getValueFromRequestParameterMap(req, ParamsNames.ADMIN_EMAIL_ID);
         Assumption.assertNotNull(emailId);
         
-        String receiverEmail = HttpRequestHelper.getValueFromRequestParameterMap(req, ParamsNames.ADMIN_EMAIL_RECEVIER);
+        String receiverEmail =  HttpRequestHelper.getValueFromRequestParameterMap(req, ParamsNames.ADMIN_EMAIL_RECEIVER);
```
Comment (wkurniawan07) [inline src/main/java/teammates/logic/automated/AdminEmailWorkerServlet.java]: Remove the extra whitespace in front of `HttpRequestHelper` there.

**Comment by gableh** [comment]
@wkurniawan07 the extra whitespace (for the first comment) seems to have been added in after i merged with master, do you want me to remove all of them?

**Comment by wkurniawan07** [comment]
@gableh try your best to. Ideally, nothing other than the fixed typo should be part of the change here.

**Comment by gableh** [comment]
alright done! sorry about that, the editor i used to resolve merge conflicts automatically removed all trailing whitespace as well.

**Comment by wkurniawan07** [comment]
@chowyb you can proceed to merge this.

**Pull request merged**